package.json

```diff
@@ -7,9 +7,13 @@
     "node": ">=6.0.0"
   },
   "scripts": {
-    "postinstall": "bash ./install.sh",
+    "postinstall": "dos2unix ./install.sh",
```

Comment:
I think this was a change for windows machine. we should try to find away to set this so it will work for windows and linux